haas/cli.py

```diff
@@ -247,17 +247,24 @@ def serve_networks():
     """Start the HaaS networking server"""
     from haas import model, deferred
     from time import sleep
+    import ConfigParser
     server.init()
     server.register_drivers()
     server.validate_state()
     model.init_db()
     migrations.check_db_schema()
+    # Check if config contains usable sleep_time
+    try:
+        sleep_time = float(cfg.get('devel', 'sleep_time'))
+    except (ConfigParser.NoSectionError, ConfigParser.NoOptionError,
+            ValueError):
+        sleep_time = 2
```

Comment:
I like @zenhack's idea of a warning, though maybe warn if it's > 1 minute, though still keeping some upper bound to prevent garbled input from being interpreted.